### ISSUE TITLE
Nice to have: Dedicated user for shell access / Better: Don't run the frontend as root

### ISSUE THREAD

**Issue by rosenrost** [emartisoft/Rastari#4]
The common pi/raspberry user should be added to be able to log in via SSH. Right now, the startup process of the frontend is part of the root user login process.

I am not so much into Hatari, but wouldn't it be better to run the system with a dedicated Rastari user (maybe the pi/raspberry one) and leave root to be what it is supposed to be? At the moment, the only way to perform superuser tasks is to call the Midnight Commander from the frontend. But you cannot open more than this one root session.

**Comment by alejandroalffer** [comment]
The only way to get root its launching a bash from Midnight Commander and then "sudo bash". If you try to change it from "raspi-config", the ncurses menu blinks and does nothing. If  you try to change the password via command line, in the next reboot the SDL interface hags after few seconds.

I'll search for a solution

**Comment by rosenrost** [comment]
A workaround might be to check the terminal before starting the frontend. That happens in ~/.profile, I think (maybe ~/.bashrc or ~/.bash_profile). If that would be restricted the one single TTY, you should be able to log in via SSH.

Another solution would be to check if the frontend is already running.

**Comment by alejandroalffer** [comment]
Definetly, all the process of launching Rastari interface it's in /root/.profile. Maybe there's any kind of autologin in the root account. I'll search deeper, but maybe, a ".deb" package of rastari can be a better solution, and not to run with "root" user, and search to do it in "pi" user. Adding a new account (midnigt commander + bash) solves the problem of accessing by ssh. Let's keep in touch: its a great interface, and having both the ".deb" and "full-distribution" as close as we can to main raspbian

**Comment by WilsonBradley** [comment]
> Definetly, all the process of launching Rastari interface it's in /root/.profile. Maybe there's any kind of autologin in the root account. I'll search deeper, but maybe, a ".deb" package of rastari can be a better solution, and not to run with "root" user, and search to do it in "pi" user. Adding a new account (midnigt commander + bash) solves the problem of accessing by ssh. Let's keep in touch: its a great interface, and having both the ".deb" and "full-distribution" as close as we can to main raspbian

Heya, not sure if you will be getting this notification after so many years.. I was going to just use Rastari img, but by now it's extremely outdated. Would rather use the new 64bit Raspberry OS build and somehow get the Rastari front end / interface on there somehow. Wish he would have made it possible to script install it..
